mmv1/products/firebase/api.yaml

```diff
@@ -152,3 +151,70 @@ objects:
           Immutable. The globally unique, Firebase-assigned identifier of the App.
 
           This identifier should be treated as an opaque token, as the data format is not specified.
+  - !ruby/object:Api::Resource
+    name: 'AndroidApp'
+    min_version: beta
+    base_url: projects/{{project}}/androidApps
+    self_link: '{{name}}'
+    update_verb: :PATCH
+    update_mask: true
+    delete_verb: :POST
+    delete_url: '{{name}}:remove'
+    description: |
+      A Google Cloud Firebase Android application instance
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/docs/android/setup'
+      api: 'https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects.androidApps'
+    async: !ruby/object:Api::OpAsync
+      actions: ["create", "delete"]
+      operation: !ruby/object:Api::OpAsync::Operation
+        path: 'name'
+        base_url: '{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'response'
+        resource_inside_response: true
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'done'
+        complete: true
+        allowed:
+          - true
+          - false
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'deletion_policy'
+        description: |
+          Optional. Set to `ABANDON` to soft-delete the AndroidApp. Soft-deleted AndroidApps are recoverable
```

Comment:
Prefer same wording as WebApp, just different default
```
(Optional) Set to `ABANDON` to allow the AndroidApp to be untracked from terraform state
          rather than deleted upon `terraform destroy`. This is useful because the AndroidApp may be 
          serving traffic. Set to `DELETE` to delete the WebApp. Default to `DELETE`
```